x-pack/plugins/apm/public/plugin.ts

```diff
@@ -135,6 +135,9 @@ export class ApmPlugin implements Plugin<ApmPluginSetup, ApmPluginStart> {
       appRoute: '/app/apm',
       icon: 'plugins/apm/public/icon.svg',
       category: DEFAULT_APP_CATEGORIES.observability,
+      meta: {
+        keywords: ['application', 'performance'],
```

Comment:
Added for testing purposes. The intent is to let plugin developers add their own keywords.